pikaday-directive.js

```diff
@@ -29,6 +29,20 @@ angular.module('pikaday', [])
                 }
             };
 
+            // Adds weekday/weekend class to cells
+            // Pikaday hides render implementation, need to consider forking it
```

Comment:
@baburdick, pikaday is handled pretty badly - 123 open issues and 84 PRs, most of them not checked. I've seen some  PRs which partially implement desired behaviour, but they were ignored. We can still add classes via ```onDraw``` method, but it allows only manipulations with already prepared DOM nodes, not the best solution.

Btw, ```pikaday-angular``` is the only package dependent on pikaday at the moment, I don't expect more dependencies on it in future.